app/assets/javascripts/OutdoorMap/map_build.js

```diff
@@ -4,106 +4,168 @@
+    crossOrigin: true,
+  }
+).addTo(mymap);
 
-L.control.locate({
+L.control
+  .locate({
     locateOptions: {
-        watch: true,
-        enableHighAccuracy: true
-    }
-}).addTo(mymap);
+      watch: true,
+      enableHighAccuracy: true,
+    },
+  })
+  .addTo(mymap);
 
 var UniPotsdamStyle = {
-        "fillColor": "Blue",
-        "fillOpacity": 0.65,
-        "color": "Blue",
-        "opacity": 0.3
-    };
+  fillColor: 'Blue',
+  fillOpacity: 0.65,
+  color: 'Blue',
+  opacity: 0.3,
+};
 
 var HPIStyle = {
-        "fillColor": "Orange",
-        "fillOpacity": 0.65,
-        "color": "Orange",
-        "opacity": 0.3
-    };
+  fillColor: 'Orange',
+  fillOpacity: 0.65,
+  color: 'Orange',
+  opacity: 0.3,
+};
 
 var DormStyle = {
-        "fillColor": "Green",
-        "fillOpacity": 0.65,
-        "color": "Green",
-        "opacity": 0.3
-    };
+  fillColor: 'Green',
+  fillOpacity: 0.65,
+  color: 'Green',
+  opacity: 0.3,
+};
 
-let layers = {}
+var EntranceStyle = {
+  fillColor: 'Red',
+  fillOpacity: 0.65,
+  color: 'Red',
+  opacity: 0.3,
+};
+
+var PoIStyle = {
+  fillColor: 'Black',
+  fillOpacity: 0.65,
+  color: 'Black',
+  opacity: 0.3,
+};
+
+let layers = {};
 
 // buildings includes all geometry-data extracted from OSM, see campus.js
 // layers has the "feature" property as index, e.g. "Studentendorf Stahnsdorfer Straße"
 for (const feature of buildings) {
-    // If the current campus (=group of buildings) is unknown, create a layergroup for it
-    if(!layers[feature.properties.campus]) {
-        layers[feature.properties.campus] = L.layerGroup().addTo(mymap);
-        // console.log("Added the following campus to layers: ", feature.properties.campus);
-    }
+  // If the current campus (=group of buildings) is unknown, create a layergroup for it
+  if (!layers[feature.properties.campus]) {
+    layers[feature.properties.campus] = L.layerGroup().addTo(mymap);
+    // console.log("Added the following campus to layers: ", feature.properties.campus);
+  }
 
-    // Determine Style (highlighting-colour) dependent of group
-    let layerStyle = UniPotsdamStyle;
-    switch(feature.properties.campus) {
-        case 'UP Campus Griebnitzsee': layerStyle = UniPotsdamStyle;
-            break;
-        case "Campus I": layerStyle = HPIStyle;
-            break;
-        case "Campus II": layerStyle = HPIStyle;
-            break;
-        case 'Campus III': layerStyle = HPIStyle;
-            break;
-        case "Studentendorf Stahnsdorfer Straße": layerStyle = DormStyle;
-            break;
-        default: console.log("This building does not belong to a known campus: ", feature);
-            break;
-    }
+  // Determine Style (highlighting-colour) dependent of group
+  let layerStyle = UniPotsdamStyle;
+  switch (feature.properties.campus) {
+    case 'UP Campus Griebnitzsee':
+      layerStyle = UniPotsdamStyle;
+      break;
+    case 'Campus I':
+      layerStyle = HPIStyle;
```

Comment:
I would suggest just using 
```
case 'Campus I':
case 'Campus II':
case 'Campus III':
 layerStyle = HPIStyle;
 break;
```
because its less code and I expect that we style these in a common look any way